.gitattributes

```diff
@@ -2,13 +2,13 @@
 # https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html
 
 # Ignore all test and documentation with "export-ignore".
-.gitattributes      export-ignore
-.gitignore          export-ignore
-.scrutinizer.yml    export-ignore
-.travis.yml         export-ignore
-CHANGELOG.md        export-ignore
-CONTRIBUTING.md     export-ignore
-docs                export-ignore
-phpunit.xml.dist    export-ignore
-README.md           export-ignore
-tests               export-ignore
+/.gitattributes      export-ignore
+/.gitignore          export-ignore
```

Comment:
Let's keep the `.gitattributes` and `.gitignore` without the slash prefix so that they are also ignored in sub directories.